Automattic/wp-calypso
# Checkout: A/B Test Automatically Displaying Coupon Input

#### Changes proposed in this Pull Request

Currently when a user enters the Checkout, they need to click "_Have a coupon code_?". This is causing issues regarding users missing this and requesting refunds, as mentioned in the original issue. As such, this PR proposes immediately displaying the coupon input.

#### Testing instructions

Switch to the `autodisplay` test

Nothing should really change except from the "Have a coupon code?" link being removed. Coupons should still be accepted and rejected the same way, and the input should still move to the bottom of the screen on mobile.

<img width="330" alt="Screenshot 2019-07-26 at 09 56 23" src="https://user-images.githubusercontent.com/43215253/61939751-dfb4e500-af8b-11e9-898e-e5716249c037.png">

The only other change is that there is no longer an autofocus set for the input, since the link has been removed.

Also verify that for the other `manualdisplay` test, everything is the same. 

cc @samouri, @adamziel, @dzver 

Fixes #31322
